docs/pages/machine-id/getting-started.mdx

```diff
@@ -288,7 +292,9 @@ ssh -F /opt/machine-id/ssh_config root@node-name.example.com
 
 <Admonition type="note" title="Roles must have logins defined">
   If you see the below error, it means the user you are trying to log in as is
-  not specified under `logins` in the role you are using.
+  not specified under `logins` in the role you are using or your role is missing
```

Comment:
This is really long sentence. I think it would be a good idea to split it at some point to make it easier to read.

Reply:
https://github.com/gravitational/teleport/pull/15117/commits/3064ed7c6b9558fe8d43c53900bd62582f610ca4